accepted/2021/statics-in-interfaces/README.md

```diff
@@ -884,22 +948,53 @@ namespace System
         static abstract TSelf Round<TInteger>(TSelf x, TInteger digits, MidpointRounding mode)
             where TInteger : IBinaryInteger<TInteger>;
 
-        // IEEE defines n to be an integral type, but not the size
+        static abstract TSelf Truncate(TSelf x);
+    }
 
-        static abstract TSelf ScaleB<TInteger>(TSelf x, TInteger n)
-            where TInteger : IBinaryInteger<TInteger>;
+    public interface IFloatingPointIeee754<TSelf>
+        : IExponentialFunctions<TSelf>,
+          IFloatingPoint<TSelf>,
+          IHyperbolicFunctions<TSelf>,
+          ILogarithmicFunctions<TSelf>,
+          IPowerFunctions<TSelf>,
+          IRootFunctions<TSelf>,
+          ITrigonometricFunctions<TSelf>
+        where TSelf : IFloatingPointIeee754<TSelf>
+    {
+        // TODO: The Exponent and Signifcand need to be exposed here
+        // They should return INumber<TSelf> here and IBinaryNumber<TSelf> on IBinaryFloatingPointIeee754<TSelf>
 
-        static abstract TSelf Sin(TSelf x);
+        // TODO: We may want to expose constants related to the subnormal and finite boundaries
 
-        static abstract TSelf Sinh(TSelf x);
+        // The following constants are defined by System.Math and System.MathF today
 
-        static abstract TSelf Sqrt(TSelf x);
+        static abstract TSelf E { get; }
 
-        static abstract TSelf Tan(TSelf x);
+        static abstract TSelf PI { get; }
 
-        static abstract TSelf Tanh(TSelf x);
+        static abstract TSelf Tau { get; }
```

Comment:
With the split, some of these constants may make sense in their own interface or as part of the `I*Functions` interfaces.

I think the expectation would be `E` belongs to log/exp and `PI`/`Tau` to to the trig functions, if not their own. Having them in their own is potentially problematic due to increased interface count.